books/tests/conftest.py

```diff
@@ -1,16 +1,40 @@
-import pytest
+import datetime
 
+import pytest
 from books.models import Book
+from django.contrib.auth.models import User
+
+from rest_framework.test import APIClient
 
 
 @pytest.fixture
 def book():
     instance = Book.objects.create(
         title="test_book",
         descriprion="test_description",
-        relase_date="2000-12-21"
-
+        phone_number=12345,
+        created_at=datetime.datetime.now()
     )
 
     return instance
 
+
+@pytest.fixture
+def user_fixture():
+    instance = User.objects.create(
+        username='test_user',
+        password='testpassword',
+        is_staff=False,
+        is_active=False,
+        is_superuser=False,
+    )
+    return instance
+
+
+@pytest.fixture
+def api_client():
+    return APIClient()
+
+@pytest.fixture
+def user_client(django_user_model, api_client):
```

Comment:
The fixture's naming is not correct here, cause it is not user client, should be smth like **auth_api_client**